componets/lists/lists.css

```diff
@@ -0,0 +1,27 @@
+.list__container {
+  align-items: center;
+  border: 1px solid var(--border-color);
+  box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
+  padding: 4px;
+}
+
+.list__heading {
+  padding-bottom: 4px;
+  border-bottom: 2px solid var(--border-color);
+}
+.list__container li {
+  list-style: none;
+  margin: 10px;
```

Comment:
Try to use global variables for properties like margin, padding and gaps, like
--space-xsmall:0.5rem

Reply:
yes I have not declared it globally from starting  so I did not added it